Update dependencies_installation.sh

QuentinVacher-rl opened the pull request:
Update versions of json and gegelati in the script, change main.cpp and update params.

QuentinVacher-rl committed:
Update dependencies_installation.sh

Update version of json and gegelati

QuentinVacher-rl committed:
Update main.cpp

Update pointers of "la.getTPGGraph"

QuentinVacher-rl committed:
Update params.json

Update params (versions with comments)

kdesnos commented on params.json:
Did you voluntarily change the param values while introducing the comments in the json file ? Or were the changed by mistake ? The commit comment does not explain the reason behind the change of values.

QuentinVacher-rl replied:
No I just copied the params file in the gegelati lib where the comments are added. I can switch to the old values, I wanted to add the commentaries to the params file.

kdesnos replied:
Yes, keep the old values when added the comments. It is important in case the selected values had be carefully handcrafted in the past. 
(You can rebase & rewrite your own history of commits if you need to keep the history clean).

QuentinVacher-rl replied:
Ok it is down. 
I set to default value the params that was not in the previous params.json

QuentinVacher-rl committed:
Update params.json

Update values of params : switch to value that was used before. Set to default value for the ones that was not registered
